### ISSUE TITLE
Following

### ISSUE THREAD

**Issue by henryl65** [raulrene/twitter-contest-js-bot#37]
Hi it seems to have an issue with following contests that require it in the tweet

**Comment by Trophonix** [comment]
Yeah, it doesn't seem like any of the things in the .then are being run actually.
